packages/block-template/README.md

```diff
@@ -1,3 +1,79 @@
-# Block Starter Kit
+# Block template
+
+## Step one: copy this template
 
 See https://blockprotocol.org/docs/developing-blocks
+
+**TL;DR:** Run `npx create-block-app \[your-block-name\]`
+
+## Step two: write and build a component
+
+1.  Change into the folder: `cd packages/hash/blocks/<name>`
+
+1.  Write a React component starting in `App.tsx`. To test it during development:
+
+    - edit `src/webpack-dev-server.js` to give your component some props to test with
+    - run the dev server with `yarn start`
+
+1.  When finished, run `yarn build`, which:
+
+    - Bundles the component, without React, into a single source file
+    - Generates a JSON schema from the `AppProps` type representing the data interface with the block
+    - Generates a `block-metadata.json` file which:
+      - points to the `schema` and `source` files
+      - brings in metadata from `package.json`, such as the block name and description
+      - lists the `externals` - libraries the block expects the host app to provide (React, unless modified)
+    - Once uploaded to a remote folder, embedding applications can access `block-metadata.json` to load a block and its schema.
+
+N.B.
+
+- The JSON schema generation assumes `AppProps` is the name of the type for the entry component's properties. If you change this name, update the `schema` script in `package.json`
+- JSON schema offers more [validations](https://json-schema.org/draft/2019-09/json-schema-validation.html) than TypeScript - **TODO** a way of storing these (e.g maxLength) in an extra config file somewhere that doesn't involve manually editing the output schema file each time
+
+## Step three: test your bundled block
+
+1.  Run `yarn serve` (when still in your `blocks/<name>` folder). Your block dist is now available at http://localhost:5000
+
+1.  In a new shell, at the root folder of this repo, run `yarn serve:hash-frontend`.
+
+1.  Visit http://localhost:3000/playground and enter http://localhost:5000 as the URL
```

Comment:
These instructions are out of date and assume the user is in a different repo. Suggest something like:

```suggestion
You can try out your block in an example embedding application:

1. Clone the example application, HASH, from [its repository](https://github.com/hashintel/hash/tree/main/packages/hash).

1. Follow the instructions in its README to set up and run HASH.

1.  In a new terminal, run `yarn serve` in your block's folder). Your block dist is now available at http://localhost:5000.

1.  In the HASH frontend (once signed in), click on the context menu next to any block on a page. Paste http://localhost:5000 into the 'load block from URL' input.

1. Your block should load in its default state. You can now test its functionality, and refresh the page to see how any changes you made persist.
```

Reply:
👍 Thanks!